handlers/bootstrap.go

```diff
@@ -125,7 +130,6 @@ func ActivateMachine(event *events.Event, apiClient *client.RancherClient) (err
 		f.WriteString(bootstrappedAt)
 		f.Close()
 	}
-	dataUpdates[bootstrappedAtField] = bootstrappedAt
```

Comment:
@wlan0 Just want to make sure I'm on the same page: You've removed the bootstrappedAtField from the resource because it is no longer needed, correct? We now key off of the files you're writing to the machine dir, correct?

If that's the case, then I'm good with this, but can you finish cleaning it up and remove all references to and declarations of `bootstrappedAtField`?